aws-ts-apigatewayv2-eventbridge/index.ts

```diff
@@ -0,0 +1,107 @@
+// Copyright 2016-2021, Pulumi Corporation.  All rights reserved.
+
+import * as aws from "@pulumi/aws";
+import * as pulumi from "@pulumi/pulumi";
+
+// Create an HTTP API.
+const api = new aws.apigatewayv2.Api("api", {
+    protocolType: "HTTP",
+});
+
+// Create a stage and set it to deploy automatically.
+const stage = new aws.apigatewayv2.Stage("stage", {
+    apiId: api.id,
+    name: pulumi.getStack(),
+    autoDeploy: true,
+});
+
+// Create an event bus.
+const bus = new aws.cloudwatch.EventBus("bus");
+
+// Create an event rule to watch for events.
+const rule = new aws.cloudwatch.EventRule("rule", {
+    eventBusName: bus.name,
+
+    // Specify the event pattern to watch for.
+    eventPattern: JSON.stringify({
+        source: ["my-event-source"],
+    }),
+});
+
+// Define a policy granting API Gateway permission to publish to EventBridge.
+const apiGatewayRole = new aws.iam.Role("api-gateway-role",
+    {
+        assumeRolePolicy: {
+            Version: "2012-10-17",
+            Statement: [
+                {
+                    Action: "sts:AssumeRole",
+                    Effect: "Allow",
+                    Principal: {
+                        Service: "apigateway.amazonaws.com",
+                    },
+                },
+            ],
+        },
+        managedPolicyArns: [
+            "arn:aws:iam::aws:policy/AmazonEventBridgeFullAccess",
+        ],
+    },
+);
+
+// Create an API Gateway integration to forward requests to EventBridge.
+const integration = new aws.apigatewayv2.Integration("integration", {
+    apiId: api.id,
+
+    // The integration type and subtype.
+    integrationType: "AWS_PROXY",
+    integrationSubtype: "EventBridge-PutEvents",
+    credentialsArn: apiGatewayRole.arn,
+
+    // The body of the request to be sent to EventBridge. Note the
+    // event source matches pattern defined on the EventRule, and the
+    // Detail expression, which just forwards the body of the original
+    // API Gateway request (i.e., the uploaded document).
+    requestParameters: {
+        EventBusName: bus.name,
+        Source: "my-event-source",
+        DetailType: "my-detail-type",
+        Detail: "$request.body",
```

Comment:
Same question as with the Python one for `$request.body.name`

Reply:
Yep, I think this is okay.